### DISCUSSION TITLE
Inserimento dei Videogame nel CMS

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#14] [General]
@agofa @bionicco @MarcoJul @SKApretto71 @zampale

Ho completato anche la parte di CMS relativa ai titoli da inserire. Queste sono le istruzioni:

1. prima occorre cercare il gioco su RAWG, qui: https://rawg.io/
   
   Immaginiamo di voler cercare Death Strending:

   ![image](https://user-images.githubusercontent.com/242049/168425578-997bbc51-0854-49c9-862a-6d4101c94598.png)

   Ottenuti i risultati si clicca sul titolo:

   ![image](https://user-images.githubusercontent.com/242049/168425625-60d281c9-b2c9-4bae-8864-b639402f31ee.png)

    e ovviamente si aprirà la scheda del gioco su RAWG. Adesso bisogna ***copiare l'ultima parte della URL del gioco***:

    ![image](https://user-images.githubusercontent.com/242049/168425726-40033a89-428c-4f8a-8279-b52633c6ec8d.png)

    Usate il copia&incolla perché deve essere esattamente come è scritto nella barra degli indirizzi.

2. dal CMS cliccate su "Aggiungi videogame":
   ![image](https://user-images.githubusercontent.com/242049/168425431-811182a5-43bf-4c61-bcfe-1610a67fab33.png)

3. si apre la scheda per l'aggiunta. Incollate lo slug che avete preso da RAWG e premete invio, senza aggiungere niente altro manualmente: il CMS va da solo su RAWG e scarica tutte le informazioni:
    ![image](https://user-images.githubusercontent.com/242049/168425787-0387364d-88e6-48fc-b9b5-3e72e6a02bd9.png)

4. se non ci credete potete a quel punto cliccare sul titolo:
    ![image](https://user-images.githubusercontent.com/242049/168425861-9e898b9c-96d8-4d5d-8786-6339e2f1975e.png)
    e scoprire che ci sono tutti i dati:
    ![image](https://user-images.githubusercontent.com/242049/168425877-f32a62d5-484f-4a1d-8bbb-640a49fada13.png)

**Comment by pdonadeo** [comment]
Ho **Completato ©®™** il primo episodio del podcast: l'interfaccia del CMS è molto molto comoda: mi sono riascoltato, facendo molti _avanti veloce_, il primo episodio e ho compilato la lista di giochi. Il risultato è qui: https://db.euscitojoypad.it/backoffice/episodio/1/change/

![image](https://user-images.githubusercontent.com/242049/168439053-e5e91583-9309-454c-81c7-d9e4cfe2182a.png)

**Reply by agofa** [reply]
WOW!
Sto facendo il secondo, in caso di semplice citazione utilizzo direi "chiacchiera libera", @pdonadeo ?

**Reply by pdonadeo** [reply]
> in caso di semplice citazione utilizzo direi "chiacchiera libera", @pdonadeo ?

Sì, l'ho messo apposta.

**Comment by agofa** [comment]
A parte che ho avuto la brillante idea di partire dall'episodio 2, ci sono decinaia di citazione visto che parla di premi, e vabbé, in alcuni casi faccio fatica a trovare il gioco giusto
Hanno citato fortnite, io ho messo quello che si chiama appunto solo così, ma c'è un fortnite 3.0
Insomma forse servirà revisione ....

**Comment by pdonadeo** [comment]
@agofa però direi che se tutti e tre parlano per diversi minuti, magari citando qualcosa d'altro come paragone, è una recensione. E magari tralascerei la citazione. Mi spiego: mentre parlo di Destiny 2 dico che ricorda in alcune dinamiche "Sparatutto A Caso III". Direi che è una recensione di Destiny 2, e "Sparatutto A Caso III" non lo citerei nemmeno, altrimenti impazziamo.

Anche perché non voglio che poi uno cerca "Sparatutto A Caso III" e lo trova, nell'episodio X, al minuto 13 e 14 secondi. Apre l'episodio, va al minuto indicato e fa perfino fatica a beccare il millisecondo in cui è stato citato il gioco. Non ha senso.

**Reply by agofa** [reply]
> Anche perché non voglio che poi uno cerca "Sparatutto A Caso III" e lo trova, nell'episodio X, al minuto 13 e 14 secondi. Apre l'episodio, va al minuto indicato e fa perfino fatica a beccare il millisecondo in cui è stato citato il gioco. Non ha senso.

In effetti hai ragione
Perché in episodio 2 fanno lista infinita di giochi candidati a qualcosa, descrivendone però di fatto solo uno
Vado a modificare

**Comment by agofa** [comment]
Ho finito episodio 2
Un altro piccolo problema è in alcuni casi capire la sintassi dei nomi pronunciati un po' così...

**Comment by bionicco** [comment]
Ottimo lavoro, ora c'è davvero tutto e anche ben funzionale!
Mi butto un po' sul data entry!
